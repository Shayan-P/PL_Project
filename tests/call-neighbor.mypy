checked def yyy(x=3, y=[1, 2, 5], z=17):
    print(x);
    print(y);
    print(z);
    print(2222);
    ;

def xxx(x=2, y=[1, 2, 3], z=12):
    yyy();
    yyy(x, y, z);
    ;

xxx(1, [1, 2], 3);
xxx();
xxx(1, [1, 77]);
