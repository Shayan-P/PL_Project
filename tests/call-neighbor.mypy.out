type-checked!
3
'(1 2 5)
17
2222
1
'(1 2)
3
2222
3
'(1 2 5)
17
2222
2
'(1 2 3)
12
2222
3
'(1 2 5)
17
2222
1
'(1 77)
12
2222
'finished
